components/aws/sagemaker/batch_transform/component.yaml

```diff
@@ -0,0 +1,21 @@
+name: 'SageMaker Batch Transformation'
+description: |
+  Batch Transformation Jobs in SageMaker
+metadata:
+  labels: {platform: 'SageMaker'}
```

Comment:
Might make sense to shift to annotations for metdata which can be freeform, if in future, and also which doesn`t have relevance for the backend Kube system

Reply:
I will just remove them. Seems not useful here.